nicework-company/unhandledexception.club
# Add time machine

## Description
Initial version of the time tool
Accessible at /tools page

## Motivation and context


## How to test
- Click on time buttons next to timezones
- Set a different time
- Check if it updated others correctly
- Try adding a new timezone
- Try updating the template text
- Try removing tokens such as `{{TIMES}}`
- Try adding timezones from the same country
- Check if flags are correct
- Try adding same times from different countries in same timezone

## Screenshots
<img width="451" alt="Screenshot 2021-02-09 at 17 27 32" src="https://user-images.githubusercontent.com/1270272/107386510-9770ff80-6afc-11eb-8a8b-df2a74532d47.png">
